scripts/update_landuse.py

```diff
@@ -42,14 +43,19 @@ def transform(self, source, result):
 mf = mule.DumpFile.from_file(restart)
 
 year = mf.fixed_length_header.t2_year
+# Expect to be updating once per year so should have month=1 and day=1
+month = mf.fixed_length_header.t2_month
+day = mf.fixed_length_header.t2_day
+if month != 1 or day != 1:
+    raise Exception(f"Unexpected month, day in update_landuse.py {month} {day}")
```

Comment:
Just a small idea but I was wondering if would be good to have a more specific exception like a `ValueError` here.

Reply:
Good idea